test/proposals/Proposal_0.sol

```diff
@@ -125,13 +125,16 @@ contract Proposal_0 is Proposal {
                 650, // midPoint = 10m50s
                 1800 // auctionDuration = 30m
             );
-            LendingTerm termUSDC1 = new LendingTerm(
+            LendingTerm termUSDC1 = new LendingTerm();
```

Comment:
LendingTerm should be created through the factory, not by calling the contract itself.

Reply:
I'll create the contracts through the factory